crates/ruff_python_parser/tests/fixtures.rs

```diff
@@ -39,7 +39,7 @@ fn test_valid_syntax(input_path: &Path) {
     });
     let parsed = parse_unchecked(&source, options);
 
-    let is_valid = parsed.is_valid() && parsed.unsupported_syntax_errors().is_empty();
+    let is_valid = parsed.has_no_errors();
```

Comment:
I think we should now rename the variable `is_valid` to match the method name. If it's only being used once we can just inline the method call there instead.